resource-watch/resource-watch
# Fixes wrong embed url in widget-block

## Overview
Fixed an issue where the generated URL for embeds was hardcoded to `/widget`, generating wrong embed URLs. Now it's generated according to the widget type. (It was broken for map widgets).

Also, moved the layout to the page.

Finally, updated some fetches.

## Testing instructions
Go to http://localhost:9000/topics/cities (production environment). Inside widget-blocks you will see a share option. The modal now renders the proper embed URL according to the type of the widget.

## Pivotal task
https://www.pivotaltracker.com/story/show/165538206

---

## Checklist before submitting
